modules/gnetBidAdapter.js

```diff
@@ -0,0 +1,111 @@
+import { registerBidder } from '../src/adapters/bidderFactory.js';
+import * as utils from '../src/utils.js';
+import { BANNER } from '../src/mediaTypes.js';
+
+const BIDDER_CODE = 'gnet';
+const ENDPOINT = 'https://grumft.cbykmedia.com/prebid/prebid.php';
+
+export const spec = {
+  code: BIDDER_CODE,
+  supportedMediaTypes: [BANNER],
+
+  /**
+   * Determines whether or not the given bid request is valid.
+   *
+   * @param {BidRequest} bid The bid params to validate.
+   * @return boolean True if this is a valid bid, and false otherwise.
+   */
+  isBidRequestValid: function (bid) {
+    return !!(bid.params.websiteId && bid.params.externalId);
+  },
+
+  /**
+   * Make a server request from the list of BidRequests.
+   *
+   * @param {validBidRequests[]} - an array of bids
+   * @return ServerRequest Info describing the request to the server.
+   */
+  buildRequests: function (validBidRequests, bidderRequest) {
+    const bidRequests = [];
+    const referer = bidderRequest.refererInfo.referer;
+
+    utils._each(validBidRequests, (request) => {
+      const data = Object();
```

Comment:
Any reason to not just init using `{}` ?

saves a couple chars for the built package.

Reply:
Done!